tests/test_OD_handler.py

```diff
@@ -0,0 +1,81 @@
+'''
+test.py
+'''
+
+import pytest
+import unittest
+import pandas as pd
+import pandas.api.types as ptypes
+
+from OD_handler.OD_handler import gr_plots, parse_data, read_xlsx, sample_outcome, gr_time_format, reshape_gr, vol_correlation, compensation_lm, gr_estimation
+
+"""The tests must run on some data, please provide an xlsx file as a result of the parsing and a file with the sample purpose named as 'calc.tsv' in the directory where the test is"""
+df_gr, df_vl = sample_outcome("calc.tsv",read_xlsx())
+df_time = gr_time_format(df_gr)
+cor_df, df_gr_cor = vol_correlation(df_time, df_vl)
+fig, df_gr_comp = compensation_lm(cor_df, df_gr_cor)
+df_gr_final = reshape_gr(df_gr_comp)
+
+
+
+class test_methods(unittest.TestCase) :
+
+ 	def test_read_parsed_xlsx_is_dataframe(self):
```

Comment:
These unit tests are a good start.  However, I recommend adding in tests for specific values within each of the data frames.  This provides a better means to test that nothing has changed in the algorithms themselves or in the handling of the initial data before it is given over the used algorithms.  

This can be done quite efficiently by generating calculated XLS files of the results that can then be read in as "expected" data frames so that you can compare each and every value in a loop.